app/lib/three_scale/swagger/specification.rb

```diff
@@ -33,9 +33,26 @@ def swagger?
         end
       end
 
+      class V30 < VBase
```

Comment:
This will require updating the tests in `test/unit/three_scale/swagger/specification_test.rb`